### PR TITLE
[lldb] [test] Disable TestTypealiasExpressionParser.py for all platforms

### PR DESCRIPTION
**Pull request by hamishknight**
This is failing on Linux https://ci.swift.org/job/oss-swift-RA-linux-ubuntu-18_04/5759/

### CODE REVIEW

**Comment by hamishknight** [comment]
@swift-ci please test

**Comment by hamishknight** [comment]
@swift-ci please test

**Comment by hamishknight** [comment]
@swift-ci please test Windows

File: lldb/test/API/lang/swift/typealias_expression_parser/TestTypealiasExpressionParser.py [hunk 1]
```diff
@@ -2,4 +2,4 @@
 from lldbsuite.test.decorators import *
 
 # FIXME! The Swift driver insists on passing -experimental-skip-non-inlinable-function-bodies-without-types to -emit-module.
-lldbinline.MakeInlineTest(__file__, globals(), decorators=[swiftTest, expectedFailureDarwin(bugnumber="rdar://120928396")])
+lldbinline.MakeInlineTest(__file__, globals(), decorators=[swiftTest, skipIf(bugnumber="rdar://120928396")])
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/typealias_expression_parser/TestTypealiasExpressionParser.py:5]:
```suggestion
lldbinline.MakeInlineTest(__file__, globals(), decorators=[swiftTest, expectedFailureAll(bugnumber="rdar://120928396")])
```

Comment (hamishknight) [reply lldb/test/API/lang/swift/typealias_expression_parser/TestTypealiasExpressionParser.py:5]: That was how I originally had it, but then it UPASSed on Ubuntu 20.04 https://ci.swift.org/job/pr-apple-llvm-project-linux/4333/

Comment (adrian-prantl) [reply lldb/test/API/lang/swift/typealias_expression_parser/TestTypealiasExpressionParser.py:5]: Right, and that is probably how this got in in the first place.

**Pull request merged**